_infra/helm/locust/locustfiles/requests.json

```diff
@@ -0,0 +1,32 @@
+{
+    "requests": [
+        {
+            "method": "GET",
+            "url": "/sign-in",
+            "expected_response_text": "Sign in"
+        },
+        {
+            "method": "POST",
+            "url": "/sign-in",
+            "data": {
+                "username": "49900000000@test.com",
+                "password": "password"
+            }
+        },
+        {
+            "method": "GET",
+            "url": "/surveys/todo",
+            "expected_response_text": "Click on the survey name to complete your questionnaire"
+        },
+        {
+            "method": "GET",
+            "url": "/surveys/history",
+            "expected_response_text": ""
+        },
+        {
+            "method": "GET",
+            "url": "/secure-message/threads",
+            "expected_response_text": "Open"
```

Comment:
Like above this doesn't seem robust enough for an assert, "Open" could be on other pages

Reply:
There is hardly anything else on the threads page that could be used to identify it exclusively as that particular page. Instead, the additional check that the Mixin does of the URL should ensure that it's looking at the correct page.

Reply:
I have added "No new conversations" as the expected text for this page.